domains/ysandeep.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ysandeepkumarreddy",
+        "email": "sandeepkumarreddy2002@gmail.com"
+    },
+    "record": {
+        "CNAME": "sandeep111.netlify.app"
```

Comment:
Why does your website require location access?

Reply:
I added a feature that shows the current weather when someone visits the webpage. It asks the user's location to do this.

Reply:
To be honest, it is pretty invasive to the users privacy. I'd like recommend you to remove that feature since we won't allow that kind of stuff (we've also banned a user for having their site ask for webcam access).